playbooks/ansible-cloud/k8s/pre.yaml

```diff
@@ -10,6 +10,11 @@
       include_role:
         name: ensure-docker
 
+    - name: Install virtualenv
+      pip:
+        name: virtualenv
+      become: true
+
```

Comment:
you can change this to ensure-virtualenv role, like we do with ensure-docker above